### DISCUSSION TITLE
Converting csv wide format MRF to JSON format

### DISCUSSION THREAD

**Discussion by MS2606** [CMSgov/hospital-price-transparency#93] [Q&A] [answered]
If we have created our MRF in csv wide format could we use the same file to convert to JSON to meet the CMS regulations? is it a correct process to create JSON MRF file? considering the csv file is in wide format and not in tall format.

**Comment by RidenourK** [comment, marked as answer]
@MS2606. Thank you for your question. 

Please review the responses in this GitHub to discussions #35 and #51, which address this question.
